analytics_data_api/v0/tests/views/test_courses.py

```diff
@@ -210,10 +210,12 @@ def assertValidActivityResponse(self, activity_type, count):
 
     @staticmethod
     def get_activity_record(**kwargs):
+        # TODO: DJ -- I'm not sure why I have to format it...
```

Comment:
I guess drf started to store datetimes as strings in the response.data instead of datetime objects.